### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by laggycomputer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,33 @@
-
 For this tutorial, we will use an Ubuntu 14 or up system (or virtual machine).
 
-ONLY WORKS ON PYTHON3.5!
+
+NOTE: This only words with Python 3.5, and **NOT PYTHON2 OR PYTHON 3.4**
```
Comment (SilBoydens) [inline README.md]: python 3.6!

**Comment by laggycomputer** [comment]
I think we should go with my version.

File: README.md (hunk 1 above)
Comment (jtagt) [reply README.md]: 🤦‍♂️ ill fix that

**Comment by jtagt** [comment]
You repeated some things. And I forgot to add that mee6 supports v1. But other than that its fine. 👍

**Pull request merged**